controller:
  name: 'right_controller'
  frame_id: 'world'
  tracking_mode: 'controller_pose'

robot:
  namespace: '' # optional
  base_frame: 'base_link'
  end_effector_frame: 'tool0'
  ft_sensor_frame: 'wrist_3_link'
  world_frame: '' # optional: if defined, apply transformation world -> robot base 

topics:
  wrench: '/wrench' # optional: wrench topic to provide haptic feedback

safety:
  play_area: [0.5, 0.5, 0.5, 25, 25, 90] # Units: meters for translation, degrees for orientation
  max_contact_force_torque: [50., 50., 50., 5., 5., 5.]
  scale_velocities: [1., 1., 1., 1., 1., 1.]
